config/spark.streaming.conf.template

```diff
@@ -16,13 +16,13 @@
 #
 
 ######
-###### This config file is a demonstration of batch processing in waterdrop config
+###### This config file is a demonstration of batch processing in seatunnel config
 ######
 
 env {
   # You can set spark configuration here
   # see available properties defined by spark: https://spark.apache.org/docs/latest/configuration.html#available-properties
-  spark.app.name = "Waterdrop"
+  spark.app.name = "Seatunnel"
```

Comment:
hi, `Seatunnel` to `SeaTunnle` is better.

Reply:
Good catch! Thanks